### ISSUE TITLE
The Recognition Accuracy on other datasets?

### ISSUE THREAD

**Issue by zobeirraisi** [opconty/Transformer_STR#3]
What is the performance on other datasets, and why only CUTE80 has been improved?

**Comment by opconty** [comment]
Because we mainly focus on curved text recognition, and we think this is
the most challenging task, and we also refine the model on this aspect.

**Comment by zobeirraisi** [comment]
Thanks for your reply, it means that after training on synthetic dataset the model again fine-tuned on CUTE-80 or real-world dataset?

**Comment by zobeirraisi** [comment]
How many epochs have been used for training?

**Comment by opconty** [comment]
Hi:

I reckon about 3 epochs for training.  Do not remember the exact number.
We do not fine-tune on real-world dataset, just pick the most skewed dataset as valid-set.
and you can train this model from scratch to figure out the result.

Best.
